GiniVision/Classes/ImageMetaInformationManager.swift

```diff
@@ -270,7 +270,8 @@ internal class ImageMetaInformationManager {
         let osVersion = UIDevice.current.systemVersion
         let giniVisionVersion = GiniVision.versionString
         let uuid = imageUUID()
-        var comment = "Platform=\(platform),OSVer=\(osVersion),GiniVisionVer=\(giniVisionVersion),\(userCommentContentId)=\(uuid)"
+        let devOrientation = deviceOrientation()
```

Comment:
I don't understand the purpose of this variable. You have a property AND a function to calculate it. I assume it is because you want to initialise the comment variable implicitly on the next line. However, what if the orientation changes in the meantime?

Reply:
I was just trying to be consistent to previous code, because it may be more clear for others to read. Yes, I thought about that possibility but It is quite unlikely to happen, isn't it?